src/components/v2v3/V2V3Project/ProjectDashboard/components/PayProjectCard/components/TokensPerEth.tsx

```diff
@@ -1,19 +1,23 @@
 import { Trans } from '@lingui/macro'
-import { useTokensPerEth } from 'components/v2v3/V2V3Project/ProjectDashboard/hooks/useTokensPerEth'
 import { V2V3CurrencyOption } from 'models/v2v3/currencyOption'
+import { V2V3_CURRENCY_ETH } from 'utils/v2v3/currency'
+import { useTokensPerEth } from '../../../hooks/useTokensPerEth'
 
 export const TokensPerEth = ({
   currencyAmount,
 }: {
   currencyAmount:
     | {
-        amount: number
+        amount: string
         currency: V2V3CurrencyOption
       }
     | undefined
 }) => {
   const { currencyText, receivedTickets, receivedTokenSymbolText } =
-    useTokensPerEth(currencyAmount)
+    useTokensPerEth({
+      amount: parseFloat(currencyAmount?.amount || '1'),
```

Comment:
account for 'no input value' here instead.

Will return as-if 1 ETH

<img width="306" alt="Screenshot 2024-01-26 at 8 12 53 AM" src="https://github.com/jbx-protocol/juice-interface/assets/12551741/73633a1d-f51d-4fd1-903c-04de7fc45651">